package.json

```diff
@@ -111,9 +111,13 @@
     "yargs": "6.6.0"
   },
   "dependencies": {
+    "angular-bootstrap-switch": "^0.5.1",
+    "angular-dragdrop": "^1.0.13",
+    "angular-ui-sortable": "^0.16.1",
     "express": "4.14.0",
     "http-proxy": "1.16.2",
-    "manageiq-ui-components": "0.0.11",
+    "jquery-ui-bundle": "^1.11.4",
+    "manageiq-ui-components": "romanblanco/ui-components#dialog_editor",
```

Comment:
Do these dependencies need to be added here? It would be great if we can keep them encapsulated in UI Components.